coverage/.resultset.json

```diff
@@ -0,0 +1,43 @@
+{
+  "RSpec": {
+    "coverage": {
+      "/home/sealfury/mid_course_project/flixcloud_api/app/channels/application_cable/channel.rb": [
+        0,
+        0,
+        0,
+        0
+      ],
+      "/home/sealfury/mid_course_project/flixcloud_api/app/channels/application_cable/connection.rb": [
+        0,
+        0,
+        0,
+        0
+      ],
+      "/home/sealfury/mid_course_project/flixcloud_api/app/jobs/application_job.rb": [
+        0,
+        null,
+        null,
+        null,
+        null,
+        null,
+        0
+      ],
+      "/home/sealfury/mid_course_project/flixcloud_api/app/models/application_record.rb": [
+        0,
+        0,
+        0
+      ],
+      "/home/sealfury/mid_course_project/flixcloud_api/app/controllers/application_controller.rb": [
+        0,
+        0
+      ],
+      "/home/sealfury/mid_course_project/flixcloud_api/app/mailers/application_mailer.rb": [
+        0,
+        0,
+        0,
+        0
+      ]
+    },
+    "timestamp": 1606847290
+  }
+}
```

Comment:
you need to add `coverage/` to your gitignore and clear your cache, you do that by running: `git rm -r --cached .` run that before you commit again with the updated gitignore